src/main/java/org/craftercms/studio/impl/v2/service/dashboard/DashboardServiceImpl.java

```diff
@@ -224,9 +219,9 @@ public ExpiringContentResult getContentExpiring(@SiteId String siteId,
     }
 
     @Override
+    @RequireSiteExists
     public ExpiringContentResult getContentExpired(String siteId, int offset, int limit)
```

Comment:
Missing `@SiteId` annotation in the parameter?

Reply:
Fixed. Thanks!